controllers/goharbor/registry/deployments.go

```diff
@@ -192,12 +192,12 @@ func (r *Reconciler) GetDeployment(ctx context.Context, registry *goharborv1.Reg
 		})
 	}
 
-	if registry.Spec.Storage.Driver.Gcs != nil && registry.Spec.Storage.Driver.Gcs.KetDataRef != "" {
+	if registry.Spec.Storage.Driver.Gcs != nil && registry.Spec.Storage.Driver.Gcs.KeyDataRef != "" {
 		volumes = append(volumes, corev1.Volume{
 			Name: "gcs-key",
 			VolumeSource: corev1.VolumeSource{
 				Secret: &corev1.SecretVolumeSource{
-					SecretName: registry.Spec.Storage.Driver.Gcs.KetDataRef,
+					SecretName: registry.Spec.Storage.Driver.Gcs.KeyDataRef,
```

Comment:
`GcsJSONKeyFilePath` is not correct, should not put in folder `/etc/registry`

Reply:
but according to the definition of helm, the files are placed in the `/etc/registry` directory
https://github.com/goharbor/harbor-helm/blob/master/templates/registry/registry-dpl.yaml#L114